test/system/610-format.bats

```diff
@@ -118,6 +92,12 @@ function check_subcommand() {
         # better error messages than just "exited with error status".
         run_podman '?' "$@" "$cmd" $extra --format '{{"\n"}}'
 
+        # Special-case workaround for quay flakes
+        if [[ "$status" -ne 0 ]] && [[ "$*" =~ search ]] && [[ "$output" =~ quay.io ]]; then
+            echo "# Ignoring probable quay flake: $output" >&3
+            continue
+        fi
```

Comment:
That is one way to deal with that :)

Reply:
I thought of simply `if search then skip this part of the test`; but I prefer having at least this minimal regression testing.